### PR TITLE
Bugfix for Issue #5, hattip to https://superuser.com/a/193638/15231

### PR DESCRIPTION
**Pull request by joereddington**
Per https://github.com/ferrine/md-img-paste.vim/issues/12

### CODE REVIEW

**Review by ferrine — COMMENTED** [review]
I'll check that soon on my laptop and respond

**Comment by ferrine** [comment]
looks like it did not break my setup

File: plugin/mdip.vim [hunk 1]
```diff
@@ -60,8 +60,12 @@ function! SaveFileTMPMacOS(imgdir, tmpname) abort
 endfunction
 
 function! SaveFileTMP(imgdir, tmpname)
-    if has('mac')
-        return SaveFileTMPMacOS(a:imgdir, a:tmpname)
+    if has('unix')
+	"from mac https://superuser.com/a/193638/15231
+	let s:uname = system("uname")
```
Comment (ferrine) [inline plugin/mdip.vim]: could you replace all tabs with spaces first?

Comment (joereddington) [reply plugin/mdip.vim]: Done :)

File: plugin/mdip.vim [hunk 2]
```diff
@@ -60,8 +60,12 @@ function! SaveFileTMPMacOS(imgdir, tmpname) abort
 endfunction
 
 function! SaveFileTMP(imgdir, tmpname)
-    if has('mac')
-        return SaveFileTMPMacOS(a:imgdir, a:tmpname)
+    if has('unix')
+      "from mac https://superuser.com/a/193638/15231
+        let s:uname = system("uname")
+            if s:uname == "Darwin\n"
```
Comment (ferrine) [inline plugin/mdip.vim:66]: What happens if this is false?

Comment (ferrine) [reply plugin/mdip.vim:66]: Maybe move this into upper level if statement? (Not sure)

**Pull request merged**